--- EmperorParser.hs	2019-07-19 23:38:59.994452470 +0100
+++ ./EmperorParser.hs2	2019-07-19 23:37:56.661393684 +0100
@@ -795,6 +795,7 @@
 emperorparserReturn1 = emperorparserReturn
 emperorparserError' :: () => ((Token), [String]) -> Alex a
 emperorparserError' tk = (\(tokens, _) -> parseError tokens) tk
+-- | Run the emperor parser
 parseEmperor = emperorparserSomeParser where
  emperorparserSomeParser = emperorparserThen (emperorparserParse 0#) (\x -> case x of {EmperorparserAbsSyn4 z -> emperorparserReturn z; _other -> notEmperorparserAtAll })
 
