--- ./parser/EmperorParser.hs.orig	2019-07-24 12:44:10.354637740 +0100
+++ ./parser/EmperorParser.hs	2019-07-24 12:44:10.358637758 +0100
@@ -841,6 +841,7 @@
 emperorparserReturn1 = emperorparserReturn
 emperorparserError' :: () => ((Token), [String]) -> Alex a
 emperorparserError' tk = (\(tokens, _) -> parseError tokens) tk
+-- | Run the emperor parser
 parseEmperor = emperorparserSomeParser where
  emperorparserSomeParser = emperorparserThen (emperorparserParse 0#) (\x -> case x of {EmperorparserAbsSyn4 z -> emperorparserReturn z; _other -> notEmperorparserAtAll })
 
