--- ./EmperorParser.hs.orig	2019-08-04 20:07:35.338583807 +0200
+++ ./EmperorParser.hs	2019-08-04 20:07:48.762367678 +0200
@@ -878,6 +878,7 @@
 emperorparserReturn1 = emperorparserReturn
 emperorparserError' :: () => ((Token), [String]) -> Alex a
 emperorparserError' tk = (\(tokens, _) -> parseError tokens) tk
+-- | Run the emperor parser
 parseEmperor = emperorparserSomeParser where
  emperorparserSomeParser = emperorparserThen (emperorparserParse 0#) (\x -> case x of {EmperorparserAbsSyn4 z -> emperorparserReturn z; _other -> notEmperorparserAtAll })
 
